### PR TITLE
Updated redhat family name in example

### PR DESCRIPTION
**Pull request by gscho**
Updated redhat family name in examples. 
'rhel' should be 'redhat'

### CODE REVIEW

File: docs/resources/os.md.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+h---
```
Comment (chris-rock) [inline docs/resources/os.md.erb]: seems like we have a small type here

**Review by chris-rock — COMMENTED** [review]
@gscho Thank you for the change. Could you please sign-off your commit? Just run 
```
git commit --amend -s
```

**Comment by chris-rock** [comment]
@gscho The content of the PR looks great. Thank you. Can we clean up the commit history a bit and rebase this on latest master? Let me know if you need any help.

**Comment by gscho** [comment]
Thanks. I squashed my commits into one but I messed up something when I fetched upstream...
I'll figure out git one day.

**Comment by chris-rock** [comment]
How I fork projects:
- fork the project
- create a branch
- add original project as separate origin (this allows you to pull master for original project easily)
- rebase own branch to master
- add PR

Let me know if I can help with anything.

**Pull request closed without merging**